wpe/src/main/cpp/browser/renderer.h

```diff
@@ -9,6 +9,9 @@ class Renderer
 public:
     virtual ~Renderer() = default;
 
+    virtual int width() const = 0;
+    virtual int height() const = 0;
```

Comment:
If those functions are simple getters, maybe it would be more accurate to call them getWidth() and getHeight()